backend/src/main/java/hu/flowacademy/vajdasagbrand/service/UserService.java

```diff
@@ -89,7 +91,7 @@ public boolean approveRegistration(String userId) throws ValidationException {
         }
     }
 
-    public void sendApprovalEmail(String email) {
-        emailService.sendMessage(email, "Registration approval", "Dear Customer! \nYour registration is approved, you can login now");
+    public void sendApprovalEmail(String email) throws MalformedURLException {
+        emailService.sendMessage(email, "Registration approval", "Dear Customer! \n \nOnce you verified your email address you will be able to login by clicking on the following link: \n" + new URL("https://iota-vajdasag-brand-b3e95c95.web.app/login") + "\n \nWelcome to Vajdasag Brand!");
```

Comment:
Ez jöhetne propertyből, configból...

Reply:
Mármint az URL-re értem.

Reply:
Sanyi kéri, hogy demora ezt most engedd be :)

Reply:
Nekem mindegy, csak egy észrevétel volt. Mivel BE-s PR, így amúgy sem én engedem majd be.